types/plugin.ts

```diff
@@ -37,3 +43,302 @@ export const Plugins: Record<PluginID, Plugin> = {
 };
 
 export const PluginList = Object.values(Plugins);
```

Comment:
Might be a good idea to remove all the no longer used variables and functions: `PluginList`, `Plugins`, `PluginName` and more from the previous plugin implementation.